justega247/favorite-things
# #167413332: Setup the landing page and routing

#### What does this PR do?
- This PR is to set up the routing for the application and the Mock landing page
#### Description of Task to be completed?
- setup routing with Vue-router
- setup the mock landing page
#### How should this be manually tested?
- Clone the project
- Navigate to the client directory
- Run **npm start**
- On your browser, navigate to **http://127.0.0.1:3000/** when the application starts
- You should see the mock homepage 
- On clicking on the get started button on the page you should be redirected to a different page
#### What are the relevant pivotal tracker stories?
[#167413332](https://www.pivotaltracker.com/story/show/167413332)
